Swamp Sneak/Assets/Standard Assets/Characters/ThirdPersonCharacter/Scripts/ThirdPersonCharacter.cs

```diff
@@ -28,7 +28,7 @@ public class ThirdPersonCharacter : MonoBehaviour
 		float m_CapsuleHeight;
 		Vector3 m_CapsuleCenter;
 		CapsuleCollider m_Capsule;
-		bool m_Crouching = false;
+		public bool m_Crouching = false;

```

Comment:
Instead of making this private variable, public. You should create public getter/setters for the variable. Then call those. That way it doesn't appear in the editor and we have more control for how the variables are set in the future.